src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -163,6 +172,12 @@ export const TransactionDetailsSummaryItemFragmentContainer = createFragmentCont
           shippingTotalCents
           taxTotal(precision: 2)
           taxTotalCents
+          buyerTotal(precision: 2)
+          buyerTotalCents
+        }
+        lastOffer {
```

Comment:
Seems like this doesn't actually use `lastOffer` right now. It probably should on the `/status` page though, but that can be another PR. For now let's just remove it?

Reply:
makes sense, i wasn't sure where it's used.